lua/apisix/schema_def.lua

```diff
@@ -273,6 +273,11 @@ local upstream_schema = {
             description = "the key of chash for dynamic load balancing",
             type = "string",
         },
+        enable_websocket = {
+            description = "enable websocket for request",
+            type        = "boolean",
+            default     = false
```

Comment:
we can delete this `default`, we can use `nil` as `false`

Reply:
sure